lib/IAMClient.js

```diff
@@ -96,7 +96,7 @@ class VaultClient {
      * @param {string} accountName - account name
      * @param {object} options - additional creation params
      * @param {string} options.email - account email
-     * @param {string} options.password - account password
```

Comment:
why do you remove options.password from the doxygen ?

Reply:
Because we do not use it anywhere as passwords are encrypted now.